.github/workflows/publish-artefact.yaml

```diff
@@ -0,0 +1,84 @@
+name: "Integration Tests and Publish on Tag"
+
+on:
+  push:
+    tags:
+    - 'autonity-v[0-9]+.[0-9]+.[0-9]+*'
+    - 'audonity-demo-v[0-9]+.[0-9]+.[0-9]+*'
+    - 'autonity-network-v[0-9]+.[0-9]+.[0-9]+*'
+
+jobs:
+  integration_tests_publish_on_tag:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: git checkout
+      uses: actions/checkout@v2
+      with:
+        repository: clearmatics/charts-ose-helm3
+        ref: helm2to3start
+
+    - name: Set useful variables
+      run: |
+        echo '::set-env name=CHART_PATH::stable/'$(echo ${GITHUB_REF##*/} | awk -F '-' '{print $1}')
+        echo '::set-env name=SHORT_SHA::'$(echo $GITHUB_SHA|head -c 7)
+        echo '::set-env name=S3_BUCKET_NAME::charts-ose.clearmatics.com'
+
+    - name: Get useful variables
+      run: |
+        echo 'Chart path: '$CHART_PATH
+        echo 'Actor: '$GITHUB_ACTOR
+        echo 'Short SHA: '$SHORT_SHA
+
+    - name: Setup gcloud environment
+      uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.2
+      with:
+        version: '290.0.1'
+        project_id: ${{ secrets.GCP_PROJECT_ID }}
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        export_default_credentials: true
+
+    - name: Configure the kubeconfig
+      run: gcloud container clusters get-credentials tf-gke-helm --zone ${{ secrets.GCP_PROJECT_ZONE }} --project ${{ secrets.GCP_PROJECT_ID }}
+
+    - name: Install helm
+      uses: azure/setup-helm@v1
+      with:
+        version: '3.2.4'
+      id: install
+
+    - name: Deploy the chart
+      run: |
+        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace $GITHUB_ACTOR
```

Comment:
Here we have to check if chart `$CHART_PATH` exist and matched with `git tag` to have a guaranty that we release correct chart/version

Reply:
Will be resolved here: https://github.com/clearmatics/charts-ose-helm3/issues/10